x-pack/plugins/reporting/server/routes/diagnostic/integration_tests/browser.test.ts

```diff
@@ -44,7 +44,12 @@ describe('POST /diagnose/browser', () => {
     httpSetup.registerRouteHandlerContext<ReportingRequestHandlerContext, 'reporting'>(
       reportingSymbol,
       'reporting',
-      () => ({ usesUiCapabilities: () => false, registerExportTypes: jest.fn() })
+      () => ({
+        usesUiCapabilities: () => false,
+        registerExportTypes: jest.fn(),
+        getSpaceId: jest.fn(),
+        getScreenshots: jest.fn(),
+      })
```

Comment:
Let's add the mock helper in `x-pack/plugins/reporting/server/mocks/index.ts`. This part of the code will be:
```
() => reportingMock.createStart()
```